pcl_conversions doesn't export PCL dependency

I'm trying to build a library using `pcl_conversions/pcl_conversions.h` via ament_cmake.
I've declared the dependency of my target to pcl_conversions

```
ament_target_dependencies(${TARGET_NAME} pcl_conversions)
```

The build fails with:

```
In file included from /path/to/cpp/file/including/pcl_conversions.cpp:N:
/opt/ros/humble/include/pcl_conversions/pcl_conversions.h:46:10: fatal error: pcl/conversions.h: No such file or directory
   46 | #include <pcl/conversions.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/pcl_conversions/CMakeLists.txt
+++ b/pcl_conversions/CMakeLists.txt
@@ -51,5 +51,5 @@ endif()
 # Export old-style CMake variables
 ament_export_include_directories("include/${PROJECT_NAME}")
 
-ament_export_dependencies(${dependencies})
+ament_export_dependencies(${dependencies} PCL)
 ament_package()

